message/agent.go

```diff
@@ -0,0 +1,3 @@
+package message
+
+const EAgent = "AGENT"
```

Comment:
This error code is undocumented and likely too generic.

Reply:
It was unused, fixed.